test/unit/util/test_get_url.py

```diff
@@ -3,6 +3,7 @@
 import responses
 
 from galaxy.util import url_get
+from galaxy_test.base.populators import skip_if_site_down
```

Comment:
This breaks the packaging structure.
`skip_if_site_down` is actually in `lib/galaxy/tool_util/unittest_utils/__init__.py` , but that would still break the packaging structure. I think we need to create `lib/galaxy/util/unittest_utils/__init__.py` and move it there (and fix the imports across the code base).

Reply:
`get_url` is a tool shed thing, maybe I can just move this into lib/galaxy/tool_shed/util ?

Reply:
@jmchilton may know ^

Reply:
it's also used in authnz (probably not necessary, but I don't want to touch everything). I don't know if I love having these unittest_utils packages everywhere ...